package.json

```diff
@@ -14,7 +14,7 @@
     "build": "gulp build",
     "test": "gulp lint test",
     "prepublish": "npm run build && npm test",
-    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i"
+    "format": "find src test | grep '\\.js$\\|\\.ts$' | xargs clang-format --style=file -i && tslint --fix -c ./tslint.json src/*.ts src/**/*.ts src/**/**/*.ts"
```

Comment:
Because without it files were left out on my machine. Looking more closely, I think it's because macOS ships an old bash.

`**` only works in bash >v4: http://unix.stackexchange.com/questions/49913/recursive-glob